js/src/ui/TxList/txList.css

```diff
@@ -42,10 +42,11 @@
     }
 
     &.timestamp {
-      padding-top: 1.5em;
-      text-align: right;
+      max-width: 5em;
+      padding-top: 2.25em;
+      text-align: center;
       line-height: 1.5em;
-      opacity: 0.5;
+      color: grey;
```

Comment:
The problem is, if opacity is dropped, the "cancel" button will also be 50% opacity LOL. Also, all font is either black or white, 0.5 opacity for both black and white revert to grey anyways.